includes/admin/customers/customer-actions.php

```diff
@@ -245,10 +246,11 @@ function edd_add_customer_email( $args = array() ) {
 		} else {
 			$redirect = edd_get_admin_url(
 				array(
-					'page'        => 'edd-customers',
-					'view'        => 'overview',
-					'id'          => urlencode( $customer_id ),
-					'edd-message' => 'email-added',
+					'page'         => 'edd-customers',
+					'view'         => 'overview',
+					'id'           => urlencode( $customer_id ),
+					'edd-message'  => 'email-added',
+					'edd-email-id' => $customer_email_id,
```

Comment:
This variable should be wrapped in `urlencode` or `absint` (since you know it's an ID I think the latter is fine, but the former would be consistent with the customer ID).